extension/ts/main/extensions/util/onLoggedIn.ts

```diff
@@ -0,0 +1,35 @@
+import {invokeWorker} from "../../../common/workers/invoker";
+import {WorkerKind} from "../../../common/workers/types";
+import {createIconButton} from "../../ui/button";
+import {isAuthorized} from "../author/util/isAuthorized";
+
+const authorize = (interactive: boolean) => invokeWorker(WorkerKind.Authorize, interactive).catch(() => "");
```

Comment:
I think the catch on this line is preventing the other catch from ever executing

Reply:
oop